client/src/components/Header/Header.js

```diff
@@ -5,10 +5,16 @@ import { MdClose } from "react-icons/md";
 import styles from "./Header.module.css";
 import { PropTypes } from "prop-types";
 
+import { instance } from "../../axios";
+
 const Sidebar = ({ onClose }) => {
   const navigate = useNavigate();
-  const handleLogOut = () => {
+  const handleLogOut = async () => {
     // TODO : API call for logout
+    const submitURL = "/api/users/logout";
```

Comment:
The `// TODO` comment as well as the `submitURL` variable can be removed since it's not used.

Reply:
Removed.